external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/HBaseLookupBolt.java

```diff
@@ -56,6 +66,30 @@ public HBaseLookupBolt withConfigKey(String configKey){
         this.configKey = configKey;
         return this;
     }
+    
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+	@Override
+    public void prepare(Map map, TopologyContext topologyContext, OutputCollector collector) {
+    	super.prepare(map, topologyContext, collector);
+    	cacheEnabled = Boolean.parseBoolean(map.getOrDefault("hbase.cache.enable", "false").toString());
```

Comment:
@HeartSaVioR sorry about that, I did see the PR build output on travis; have it on my TODO list.